api-client/src/main/java/com/desk/java/apiclient/model/OpportunityAttachment.java

```diff
@@ -26,24 +26,19 @@
 
 package com.desk.java.apiclient.model;
 
-import com.google.gson.annotations.SerializedName;
-
-import java.io.Serializable;
+import java.util.Date;
 
 /**
  * Created by Matt Kranzler on 1/14/16.
  * Copyright (c) 2016 Desk.com. All rights reserved.
  */
-public class OpportunityAttachment extends Attachment implements Serializable {
-
-    private static final long serialVersionUID = 2633496287407407208L;
+public class OpportunityAttachment extends OpportunityActivity {
```

Comment:
Why does this extend `OpportunityActivity` now? The reason it extended `Attachment` was because it was the same model in the backend with a few different fields. If there isn't a good reason I think this should just implement `IOpportunityActivity`.